index.js

```diff
@@ -40,8 +41,11 @@ cluster.on('exit', function (worker) {
 });
 
 if (cluster.isMaster) {
-    const cpuCount = os.cpus().length;
+    cpuCount = os.cpus().length;
     for (let j = 0; j < cpuCount; j++) {
+        if (FORKS > 0) {
+            cpuCount = FORKS;
+        }
```

Comment:
That `cpuCount` needs a `let`. Also, the `if (FORKS > 0) [...]` lines should be above the `for`. I'll merge and amend this afterwards